PlateSpace/ETL/clearRestaurantsCollection.js

```diff
@@ -0,0 +1,21 @@
+var mongoose = require('mongoose');
```

Comment:
I'm inclined to say this file isn't needed either. If we really want to provide the user with a way to clear that data, then perhaps we could accept a parameter to `index.js` on the command line (e.g. `clear` vs `init`)?